packages/assets/css/src/scss/_components/_checkbox/_elements/_checkbox-box.scss

```diff
@@ -16,7 +16,8 @@
 			@include element(box) {
 				width: $checkbox-size-md;
 				height: $checkbox-size-md;
-				display: flex;
+				display: flex !important;
```

Comment:
@abder Is '!important' required can we try not to use it?

Reply:
@emgk there's a style set by common styles in: .storybook/assets/css/wordpress/common.css which I wanted to override